model/feature_flags.go

```diff
@@ -101,6 +104,8 @@ func (f *FeatureFlags) SetDefaults() {
 	f.ReduceOnBoardingTaskList = false
 	f.ThreadsEverywhere = false
 	f.GlobalDrafts = true
+	f.GlobalDrafts = true
```

Comment:
This line is duplicated

Reply:
fixed ... was a mistake while adding the default value for the WYSIWYG flag